samples/gltf_viewer.cpp

```diff
@@ -101,6 +101,7 @@ struct App {
 
     bool actualSize = false;
     bool originIsFarAway = false;
+    float originDistance = 6378137;
```

Comment:
Comment to explain this value?

Reply:
isn't it obvious it's the radius of Earth in meters? :-)